### PR TITLE
Add environment.yml and requirements.txt for Tingley septal conversion

### PR DESCRIPTION
**Pull request by h-mayorquin**
This is to discuss how (or if) we should store the `requirements.txt` (for pip) and the `environment.yml` (for conda). I added the frozen version of the state of the repos just after I ran the conversion. 

Right now, they just stand as an indicator of what libraries the environment that I used in dandi had. I am not sure this will be sufficient to reproduce the environment as I have not tested it but maybe this should be enough?

Let me know what you think.

### CODE REVIEW

File: buzsaki_lab_to_nwb/tingley_code_septal/tingley_septal_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+asciitree==0.3.3
+attrs==21.4.0
+backcall @ file:///home/ktietz/src/ci/backcall_1611930011877/work
+Bottleneck @ file:///tmp/build/80754af9/bottleneck_1607575130224/work
```
Comment (CodyCBakerPhD) [inline buzsaki_lab_to_nwb/tingley_code_septal/tingley_septal_requirements.txt:4]: What are all these @ items? Are they really necessary?

**Comment by CodyCBakerPhD** [comment]
In general I would think all we really need is (a) version of `nwb-conversion-tools`, and (b) version of `spikeextractors`. Those are the API's that have the most changes related to debugs, function calls, etc.

While the full frozen state would indeed be requires to perfectly (100%) ensure reproducibility of original files, this might be overkill for our purposes.

**Comment by h-mayorquin** [comment]
> In general I would think all we really need is (a) version of `nwb-conversion-tools`, and (b) version of `spikeextractors`. Those are the API's that have the most changes related to debugs, function calls, etc.
> 
> While the full frozen state would indeed be required to perfectly (100%) ensure reproducibility of original files, this might be overkill for our purposes.

@CodyCBakerPhD 
The changes that we used in both  `nwb-conversion-tools` and `spikeextractors` are not included in any tagged version of them, aren't they? (i.e. the latest additions to `cellexplorersortingextractor` in spikeextractors).

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin No, probably not; you can pin it to the commit hash of the version state used successfully to run the conversion though (example: https://github.com/catalystneuro/nwb-conversion-tools/blob/main/requirements-full.txt#L13-L14)

File: buzsaki_lab_to_nwb/tingley_code_septal/tingley_septal_requirements.txt (hunk 1 above)
Comment (h-mayorquin) [reply buzsaki_lab_to_nwb/tingley_code_septal/tingley_septal_requirements.txt]: They point directly to the build version in the local system which is redundant. Let's go with the minimal pinned version.

**Comment by h-mayorquin** [comment]
> @h-mayorquin No, probably not; you can pin it to the commit hash of the version state used successfully to run the conversion though (example: https://github.com/catalystneuro/nwb-conversion-tools/blob/main/requirements-full.txt#L13-L14)

@CodyCBakerPhD 
The latest commit follows your suggestion above to only have the pinned versions of `nwb-conversion-tools` and `spikeextractors`.

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin Looks great!

**Pull request merged**